modules/ding_provider/ding_provider.module

```diff
@@ -227,13 +237,6 @@ function ding_provider_invoke_page($type, $hook) {
       // Rethrow user exceptions.
       throw $e;
     }
-    elseif ($e instanceof DingProviderAuthException) {
```

Comment:
It's an old left over that we never hit (checks are made before calling part of the code). It would redirect to "user/login" which will redirect to the front page of the site (without opening the login box).

Reply:
Then it should be cleaned up. The issue here is that at this place in the code have no way of knowning what should happen.

Where should the user be sendt to (in the "new" ddbasic the should be `<front>#login`) which would remove the "destination" given in https://github.com/ding2/ding2/blob/master/modules/ding_provider/ding_provider.module#L234 but also the `#login` don't work anymore for some reason).

With adgangsplaformen this should be a redirect and back as the `user/login` is restored to allow non-provider users to login. Here I could detected if the `ding_adgangsplatformen` exists and do the same as in `ding_adgangsplatformen_login` (with yet a login wrapper function).

Need some more input here. My solution clean up the documentation... I don't see any code currently calling `ding_provider_invoke_page` which is not in context where the user is not logged in.

Reply:
Restored function with deprecated warning about the issues.